### PR TITLE
chore(onboarding-1.5): instrumentation and 1.0 flag removal

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

- remove pre 1.0 onboarding ingestion wizard UI code
- update with instrumentation for 1.5

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/ingestion/ingestionLogic.ts [hunk 1]
```diff
@@ -125,18 +125,10 @@ export const ingestionLogic = kea<ingestionLogicType>([
                 return (verify ? 1 : 0) + (framework ? 1 : 0) + (platform ? 1 : 0)
             },
         ],
-        onboarding1: [
-            () => [],
-            (): boolean => {
-                const featFlags = values.featureFlags
-                return featFlags[FEATURE_FLAGS.ONBOARDING_1] === 'test'
```
Comment (EDsCODE) [inline frontend/src/scenes/ingestion/ingestionLogic.ts:132]: This flag constant can be cleaned up

Comment (liyiy) [reply frontend/src/scenes/ingestion/ingestionLogic.ts:132]: ooop good catch

File: posthog/tasks/email.py [hunk 2]
```diff
@@ -148,6 +149,7 @@ def send_first_ingestion_reminder_emails() -> None:
 
                 message.add_recipient(user.email)
                 message.send()
+                report_first_ingestion_reminder_email_sent()
```
Comment (liyiy) [inline posthog/tasks/email.py:152]: @neilkakkar these look okay?

File: posthog/event_usage.py [hunk 3]
```diff
@@ -206,6 +206,22 @@ def report_organization_deleted(user: User, organization: Organization):
...
 
+def report_first_ingestion_reminder_email_sent(user: User):
+    posthoganalytics.capture(
+        user.distinct_id,
+        "first ingestion reminder email sent",
+        groups={"organization": str(user.current_organization.id), "instance": SITE_URL,},
+    )
+
+
+def report_second_ingestion_reminder_email_sent(user: User):
```
Comment (EDsCODE) [inline posthog/event_usage.py:217]: NIT: Thoughts on parametrizing this? Repetitive structure that would eventually get refactored

**Review by EDsCODE — APPROVED** [review]
Last NIT otherwise lgtm

File: posthog/event_usage.py (hunk 3 above)
Comment (liyiy) [reply posthog/event_usage.py:217]: Yeah I thought about that when I wrote it but I think having it as a separate event will make it easier to measure/track on the app? 😁

Comment (liyiy) [reply posthog/event_usage.py:217]: and if we end up having more ingestion reminder emails we should definitely refactor it !

**Comment by neilkakkar** [comment]
Curious why we don't simply use actions here? 🤔

**Pull request merged**